openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -141,7 +140,8 @@ def set_logged_in_cookies(request, response, user):
 
         _set_deprecated_logged_in_cookie(response, cookie_settings)
         set_deprecated_user_info_cookie(response, request, user, cookie_settings)
-        _create_and_set_jwt_cookies(response, request, cookie_settings, user=user)
+        if settings.FEATURES.get('ENABLE_LOGIN_SET_JWT_COOKIES', True):
```

Comment:
Reviewers FYI: 
1. This is an update where I protect this call to `_create_and_set_jwt_cookies`, but not the other call from inside `refresh_jwt_cookies`.  I also renamed the feature flag by adding the term `LOGIN`.
2. Although it is slightly awkward to have this test-only feature flag, I prefer the flag being named such that you can understand what is being disabled.  An alternative would be adding a setting like `IN_UNIT_TESTS` (or something like this), and then you'd have really awkward unit tests that disable `IN_UNIT_TESTS` to really test this functionality.